apispec/openapi-circe-yaml/src/main/scala/sttp/tapir/openapi/circe/yaml/TapirOpenAPICirceYaml.scala

```diff
@@ -1,12 +1,15 @@
 package sttp.tapir.openapi.circe.yaml
 
-import sttp.tapir.openapi.OpenAPI
-import sttp.tapir.openapi.circe._
 import io.circe.syntax._
 import io.circe.yaml.Printer
+import io.circe.yaml.Printer.StringStyle
+import sttp.tapir.openapi.OpenAPI
+import sttp.tapir.openapi.circe._
 
 trait TapirOpenAPICirceYaml {
   implicit class RichOpenAPI(openAPI: OpenAPI) {
     def toYaml: String = Printer(dropNullKeys = true, preserveOrder = true).pretty(openAPI.asJson)
+    def toYaml(stringStyle: StringStyle): String =
```

Comment:
maybe add the same to `TapirAsyncAPICirceYaml`?

Reply:
right, added